.github/work/linters.yml

```diff
@@ -0,0 +1,62 @@
+name: Linters
```

Comment:
- [ ] After the linters, solve all issues that this could have. 🐱

Reply:
I'm stuck in the part of linters... :c

Reply:
I don't get the idea about how to check for issues in my code

Reply:
I've been reading the documentation but I can't understand

Reply:
The workflows tell me that there are no errors